### PR TITLE
Add close applications behaviour

### PR DESCRIPTION
**Pull request by casassg**
Allow for applications to be closed (and close it for now)

### CODE REVIEW

File: app/settings.py [hunk 1]
```diff
@@ -241,9 +241,10 @@ def ACCOUNT_USER_DISPLAY(x):
     'r_organizer_email': '^.*@hackupc\.com$',
     'google_analytics': 'UA-69542332-2',
     'when_to_arrive': 'Registration opens at 3:00 PM and closes at 6:00 PM on Friday October 13th, '
-    'the opening ceremony swill be at 7:00 pm.',
+                      'the opening ceremony swill be at 7:00 pm.',
```
Comment (AsLogd) [inline app/settings.py]: swill

**Review by AsLogd — APPROVED** [review]

File: app/settings.py (hunk 1 above)
Comment (casassg) [reply app/settings.py]: thanks! Fixed!

File: app/templates/include/subscribe_form.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 {% settings_value "STATIC_KEYS_TEMPLATES" as static_values %}
 
 {% if static_values.subscribe_url %}
-    Subscribe here to be the first to know when are we open applications for next edition.
+    <p>Subscribe here to be the first to know when are we open applications for next edition.</p>
```
Comment (wailingtam) [inline app/templates/include/subscribe_form.html]: Subscribe here to be the first to know when `applications for the next edition are open`.

File: register/templates/apply.html [hunk 3]
```diff
@@ -3,9 +3,11 @@
 {% block head_title %}Apply{% endblock %}
 {% block body %}
     {% settings_value 'REGISTER_APP' as app_data %}
-    {% if user.hacker %}
+    {% if not applications_open %}
+        <h4 class="text-center margin-top" style="color: black;">Applications are closed. We are looking forward to welcome you next edition.</h4>
```
Comment (wailingtam) [inline register/templates/apply.html]: Applications are closed. We are looking forward to welcome you `on the` next edition.

File: register/templates/phases/application.html [hunk 4]
```diff
@@ -1,4 +1,12 @@
-{% if not application and user.hacker %}
+{% load utils %}
+
+{% settings_value "STATIC_KEYS_TEMPLATES" as static_values %}
+
+{% if not application and not applications_open %}
+    <p>Applications have been closed.  We are looking forward to welcome you next edition.</p>
```
Comment (wailingtam) [inline register/templates/phases/application.html]: Same. 
...`on the` next edition

File: register/templates/phases/profile.html [hunk 5]
```diff
@@ -1,9 +1,18 @@
-<p>You will only need to fill this once, we will keep this information between editions
-{% if user.hacker %}
-You can edit it as well.
-{% endif %}
+{% load utils %}
+
+{% settings_value "STATIC_KEYS_TEMPLATES" as static_values %}
+{% if not user.hacker and not applications_open %}
+    <p>Applications have been closed. We are looking forward to welcome you next edition.</p>
```
Comment (wailingtam) [inline register/templates/phases/profile.html]: same

**Review by wailingtam — CHANGES_REQUESTED** [review]
❌

**Comment by casassg** [comment]
@wailingtam fixed

**Review by wailingtam — APPROVED** [review]

**Pull request merged**